diff --git a/hosts/default/configuration.nix b/hosts/default/configuration.nix
index 42a335f..4de99d9 100644
--- a/hosts/default/configuration.nix
+++ b/hosts/default/configuration.nix
@@ -153,6 +153,7 @@
     gnome.gnome-disk-utility
     gparted
     gnumake
+    gnupg
     hackrf
     hollywood
     htop
@@ -194,6 +195,7 @@
     obs-studio
     openscad # Funny code-based 3D design tool (thanks Steve <3)
     pavucontrol # PulseAudio Volume Controls
+    pinentry # Authentication for GPG
     plasma-systemmonitor
     prismlauncher # Alternative Minecraft launcher
     #posy-cursors # TODO Contribute this package perhaps?

